### PR TITLE
Public Preview Refresh Documentation

### PR DESCRIPTION
**Pull request by OWinfreyATL**

### CODE REVIEW

**Comment by OWinfreyATL** [comment]
This PR contains addressed feedback from https://github.com/microsoftgraph/microsoft-graph-docs/pull/20058

File: api-reference/beta/resources/identitygovernance-lifecycleworkflowscontainer.md [hunk 1]
```diff
@@ -45,7 +45,6 @@ The following is a JSON representation of the resource.
   "blockType": "resource",
   "keyProperty": "id",
   "@odata.type": "microsoft.graph.identityGovernance.lifecycleWorkflowsContainer",
-  "baseType": "microsoft.graph.entity",
```
Comment (msewaweru) [inline api-reference/beta/resources/identitygovernance-lifecycleworkflowscontainer.md:48]: Confirm this inheritance.

File: api-reference/beta/resources/identitygovernance-run.md [hunk 2]
```diff
@@ -35,12 +33,14 @@ Inherits from [entity](../resources/entity.md).
...
 |processingStatus|String|The run execution status. The possible values are: `queued`, `inProgress`, `completed`, `completedWithErrors`, `canceled`, `failed`, `unknownFutureValue`.<br><br>Supports `$filter`(`eq`, `ne`) and `$orderby`.|
 |startedDateTime|DateTimeOffset|The date time that the run execution started.<br><br>Supports `$filter`(`lt`, `le`, `gt`, `ge`, `eq`, `ne`) and `$orderby`.|
 |scheduledDateTime|DateTimeOffset|The date time that the run is scheduled to be executed for a workflow.<br><br>Supports `$filter`(`lt`, `le`, `gt`, `ge`, `eq`, `ne`) and `$orderby`.|
 |successfulUsersCount|Int32|The number of successfully completed users in the run.|
 |totalUsersCount|Int32|The total number of users in the workflow execution.|
+|totalTasksCounts|Int32|The total number of tasks in the run execution.|
 |totalUnprocessedTasksCount|Int32|The total number of unprocessed tasks in the run execution.|
 |workflowExecutionType|String|The execution type of the workflows associated with the run. The possible values are: `scheduled`, `onDemand`, `unknownFutureValue`.<br><br>Supports `$filter`(`eq`, `ne`) and `$orderby`.|
```
Comment (msewaweru) [inline api-reference/beta/resources/identitygovernance-run.md:45]: Correct the type of this property.

File: api-reference/beta/resources/identitygovernance-run.md [hunk 3]
```diff
@@ -35,12 +33,14 @@ Inherits from [entity](../resources/entity.md).
 |completedDateTime|DateTimeOffset|The date time that the run completed. Value is `null` if the workflow hasn't completed.<br><br>Supports `$filter`(`lt`, `le`, `gt`, `ge`, `eq`, `ne`) and `$orderby`.|
 |failedTasksCount|Int32|The number of tasks that failed in the run execution.|
 |failedUsersCount|Int32|The number of users that failed in the run execution.|
+|id|String|A unique identifier for the workflow run.|
 |lastUpdatedDateTime|DateTimeOffset|The datetime that the run was last updated.<br><br>Supports `$filter`(`lt`, `le`, `gt`, `ge`, `eq`, `ne`) and `$orderby`.|
 |processingStatus|String|The run execution status. The possible values are: `queued`, `inProgress`, `completed`, `completedWithErrors`, `canceled`, `failed`, `unknownFutureValue`.<br><br>Supports `$filter`(`eq`, `ne`) and `$orderby`.|
```
Comment (msewaweru) [inline api-reference/beta/resources/identitygovernance-run.md:38]: As commented above. Return type ss property is wrong.

**Review by msewaweru — CHANGES_REQUESTED** [review]
@OWinfreyATL, thanks for the work on this PR. The highlights of the review are as below. I will complete reviewing the change log on Monday. Thanks.
1. Resolve the build warning on this PR.
2. Maintain indentation on the files you updated to remove it. It hugely affects readability.
3. Confirm the updates that are removing the  baseType entity in the JSON file.

File: api-reference/beta/resources/identitygovernance-run.md (hunk 2 above)
Comment (OWinfreyATL) [reply api-reference/beta/resources/identitygovernance-run.md:45]: Im seeing the same type as totalUsersCount:
image

File: changelog/Microsoft.AAD.LCM.json [hunk 4]
```diff
@@ -350,6 +350,450 @@
...
+      ],
+      "Id": "c34adaa0-b7de-40d7-92a4-1abd8e37400a",
+      "Cloud": "Prod",
+      "Version": "v1.0",
+      "CreatedDateTime": "2023-02-02T05:06:14.6127999Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Governance"
+    },
     {
       "ChangeList": [
```
Comment (msewaweru) [inline changelog/Microsoft.AAD.LCM.json:798]: @OWinfreyATL - This change log replicates the v1 change log that is currently on hold. Please reset the change log to only capture the emailsettings change (download the change log from Drake's PR) and extensibility resources change.

**Review by msewaweru — CHANGES_REQUESTED** [review]
@OWinfreyATL, I have completed the review on the change log. See my comments and update. Thanks.

**Comment by msewaweru** [comment]
@FaithOmbongi - as discussed, I have reviewed the changes and all is ok except the pending items, change log and [this comment](https://github.com/microsoftgraph/microsoft-graph-docs/pull/20510#discussion_r1140531077) that I marked as unresolved.

**Review by FaithOmbongi — APPROVED** [review]
Hi @OWinfreyATL - This looks good to go. Approved.

@msewaweru - Thank you for your extensive review of this PR.

**Pull request merged**